backend/src/Designer/Services/Implementation/SourceControlLoggingDecorator.cs

```diff
@@ -170,6 +171,15 @@ public void FetchRemoteChanges(string org, string repository)
             {
                 _decoratedService.FetchRemoteChanges(org, repository);
             }
+            catch (LibGit2Sharp.LibGit2SharpException libGit2SharpException)
+            {
+                if (libGit2SharpException.Message.Contains("server requires authentication that we do not support"))
```

Comment:
I assume this is not related only to the fetchRemoteChanges method. Can this be also handled globally in filter?

Reply:
Fixed ✅